### ISSUE TITLE
HS 200 Tp link not connecting to Homebridge

### ISSUE THREAD

**Issue by jayhabs** [plasticrake/homebridge-tplink-smarthome#238]
### Versions

- **Node**:       `node --version`
- **Homebridge**: `homebridge --version`
- **OS**:         

### Configuration

```json

```

### Homebridge Log / Command Output

```

Hello I am running the current version of Homebrige with Node.js Version | v16.13.2 -NPM version v8.1.2.

I am trying to install the Kasa TP-link HS 200 switch.  The plugin is not recognizing the switch.  Could someone provide a step by step guide to the device discovery section in the plugin or any other suggestions on how to gain access to this switch in HomeKit 

Thanks everyone


```

**Comment by EricSabe** [comment]
Is Homebridge sitting on the same network as the TP-Link device?  Does that network permit broadcast traffic? Kasa uses broadcast to discover devices.  Alternatively, you can use the Manual Discovery section in the plug-in's config to specific the IP addresses of your Kasa devices.

**Comment by jayhabs** [comment]
Yes they are the same network, where do you find the specific IP address, is it on the network router settings, thanks

**Comment by EricSabe** [comment]
On whatever is assigning IP addresses (that could be your router), it will typically be found in a view that includes DHCP Leases.  If Manual Discovery is your fix, you'll want to create a DHCP Reservation for each TP-Link device to prevent the IP address from changing.

**Comment by JohnnySoprano** [comment]
Were you able to figure it out ? Im having the same issue

**Comment by dwilasnd** [comment]
I have HS210 that is recognized, but my HS200 do not get discovered.
